### ISSUE TITLE
More languages

### ISSUE THREAD

**Issue by Haydyn1551** [CodePadSite/codepad-issues#66]
Its a little annoying to make a game/website without a  YAML and epk files and it would be cool to add C+++ for game development.

**Comment by ethan-vanderheijden** [comment]
> it would be cool to add C+++ for game development.

Yes, I definitely agree. Supporting more languages (#67, #68, #69, #70, #71) has been on our radar for a long time, but we never got around to it. I think we should make some progress on this within the next few months.

> Its a little annoying to make a game/website without a YAML and epk files

Don't we support YAML files already? YAML files have a file icon and support syntax highlighting. If there is any functionality missing, let us know.

I'm not familiar with .epk files. What are they used for and what does a typical one look like?

**Comment by stevenwhiz** [comment]
What game library are you interested in using?

While you can't create a C++ pad, you can compile and run simple C++ programs in CodePad.  Create a Java pad and create a new C++ program (e.g. main.cpp).  Then bring up a shell by clicking the "+" next to Console and select "Shell".  You can then compile and run the program as follows:

g++ main.cpp -o main
./main

**Comment by Haydyn1551** [comment]
A good game library for C++ would probably be SFML and also I found out
codepad has a file that ends in .sh (i don't know the name of the language)
and I didnt yaml was already in codepad.
